app/services/PaymentServices.scala

```diff
@@ -23,18 +24,28 @@ class PaymentServices(
   actorSystem: ActorSystem
 )(implicit ec: ExecutionContext) {
 
-  val stripeServices: Map[PaymentMode, StripeService] = {
+  import utils.ConfigUtils._
+  import utils.FastlyUtils._
+
+  def stripeServices(maybeCountryGroup: Option[CountryGroup]): Map[PaymentMode, StripeService] = {
```

Comment:
as discussed, preferable not to create a new instance of the map each time this function is invoked